examples/chef/linux/main.cpp

```diff
@@ -29,6 +29,19 @@ using namespace chip::Shell;
 using namespace chip::app;
 using namespace chip::app::Clusters;
 
+#if CHIP_CONFIG_ENABLE_ICD_CIP
+// Wake up periodically to send Check-in message since Linux application doesn't have physical button
+void notifyIcdActive(System::Layer * layer, void *)
+{
+    ICDNotifier::GetInstance().NotifyNetworkActivityNotification();
+
+    ChipLogDetail(Shell, "Periodic wakeup to notify ICD active");
+    DeviceLayer::SystemLayer().StartTimer(
+        chip::System::Clock::Milliseconds32(LinuxDeviceOptions::GetInstance().icdPeriodicWakeupDurationMs), notifyIcdActive,
```

Comment:
This can only be sent if the device has no subscriber.

Reply:
Removed